src/modules/keyboardmanager/KeyboardManagerEditorLibrary/KeyDropDownControl.cpp

```diff
@@ -379,7 +379,14 @@ void KeyDropDownControl::SetDropDownError(ComboBox currentDropDown, hstring mess
 {
     currentDropDown.SelectedIndex(-1);
     warningMessage.as<TextBlock>().Text(message);
-    currentDropDown.ContextFlyout().ShowAttachedFlyout((FrameworkElement)dropDown.as<ComboBox>());
+    try
+    {
+        currentDropDown.ContextFlyout().ShowAttachedFlyout((FrameworkElement)dropDown.as<ComboBox>());
+    }
+    catch (winrt::hresult_error const&)
+    {
+        // If it's loading and some remaps are invalid from previous configs, avoid crashing when flyouts can't be showed yet.
```

Comment:
Let's log a warning here. It would be useful if there is a remapping that can be added but can not be displayed.

Reply:
I've pushed the requested changes.